bazel/glog.bzl

```diff
@@ -56,6 +56,7 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
     linux_or_darwin_copts = wasm_copts + [
         # For src/utilities.cc.
         "-DHAVE_SYS_SYSCALL_H",
+        "-DHAVE_UNISTD_H",
```

Comment:
Could you add a comment `# For -log_link.`? `For src/utilities.cc.` is not so relevant here.

Reply:
Done